flang/test/Lower/Intrinsics/perror.f90

```diff
@@ -0,0 +1,45 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck --check-prefixes=CHECK %s
+! RUN: %flang_fc1 -emit-fir %s -o - | FileCheck --check-prefixes=CHECK %s
```

Comment:
Ahh sorry I didn't notice this before, please could you use `-emit-hlfir` and regenerate the test checks?

We test lowering to HLFIR separately from conversion from HLFIR to FIR so that each lit test is only checking one thing at a time. It makes it easier to see how the code changes added here are reflected in the pipeline if we aren't looking at a version that already had several complex passes run on it.